master/getting-started/bare-metal/bare-metal.md

```diff
@@ -39,17 +39,13 @@ Calico supports selector-based policy as normal when running on a gateway or rou
 allowing for rich, dynamic security policy based on the labels attached to your
 workloads.
 
-> **NOTE**
->
-> If you have a host with workloads on it then traffic that is forwarded to
-> workloads bypasses the policy applied to host endpoints. If that weren't the
-> case, the host endpoint policy would need to be very broad to allow all
-> traffic destined for any possible workload.
->
-> Since version 2.1.0, Calico applies host endpoint policy to traffic that is
-> being forwarded between host interfaces.
->
-> ![]({{site.baseurl}}/images/bare-metal-packet-flows.png)
+![]({{site.baseurl}}/images/bare-metal-packet-flows.png)
+
+
+<div class="alert alert-info" role="alert"><b>Note</b>: If you have a host with workloads on it then traffic that is forwarded to workloads bypasses the policy applied to host endpoints. If that weren't the case, the host endpoint policy would need to be very broad to allow all traffic destined for any possible workload. 
+<p></p>
+<p></p>
```

Comment:
Extra empty para here?

Reply:
Actually, seems like there are lots more of these `<p></p>` occurrences below - are they really needed?

Reply:
This is the way to get the proper line breaks in the `<div>` elements: `<p></p><p></p>`